XCUITests/DesktopModeTests.swift

```diff
@@ -3,7 +3,28 @@
  file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import XCTest
-class DesktopModeTests: IphoneOnlyTestCase {
+
+// Tests for both platforms
+class DesktopModeTests: BaseTestCase {
+    func testLongPressReload() {
+        navigator.openURL(path(forTestPage: "test-user-agent.html"))
+        waitUntilPageLoad()
+        XCTAssert(app.webViews.staticTexts.matching(identifier: "MOBILE_UA").count > 0)
+
+        navigator.goto(ReloadLongPressMenu)
+        navigator.performAction(Action.ToggleRequestDesktopSite)
+        waitUntilPageLoad()
+        XCTAssert(app.webViews.staticTexts.matching(identifier: "DESKTOP_UA").count > 0)
+
+        navigator.performAction(Action.CloseTabFromTabTrayLongPressMenu)
+
+        navigator.openURL(path(forTestPage: "test-user-agent.html"))
+        waitUntilPageLoad()
+        XCTAssert(app.webViews.staticTexts.matching(identifier: "DESKTOP_UA").count > 0)
+    }
+}
+
+class DesktopModeTestsIphone: IphoneOnlyTestCase {
```

Comment:
Could you disable this test suite from the `Fennec_Enterprise_SmokeXCUITests` schema, please? Now it is enable there and in `Fennec_Enterprise_XCUITests` and  we will have the tests running twice...